fiftyone/server/metadata.py

```
@@ -118,6 +119,10 @@ async def get_metadata(
                     aspect_ratio=width / height,
                 )
 
+    if filepath not in metadata_cache and skip_dimensions:
+        # caller opted out of dimensions; use a placeholder aspect ratio
+        metadata_cache[filepath] = dict(aspect_ratio=1.0)
```

Ruff (0.15.18)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)